app/Domain/CalendarGenerator.php

```diff
@@ -42,6 +42,7 @@ protected function generateCalendar(CarbonPeriod $period, Collection $holidays):
                 "isToday" => $day->isToday(),
                 "isWeekend" => $day->isWeekend(),
                 "isHoliday" => $holidays->contains($day),
+                "isFuture" => $day->isToday() || $day >= Carbon::now(),
```

Comment:
```suggestion
                "isFuture" => $day->isFuture(),
```
Maybe this would be more readable?